server/src/test/java/org/elasticsearch/action/search/MultiSearchRequestTests.java

```diff
@@ -537,6 +538,14 @@ public void testFailOnExtraCharacters() throws IOException {
         }
     }
 
+    public void testNullIndex() throws IOException {
+        try {
+            MultiSearchRequest request = parseMultiSearchRequest("/org/elasticsearch/action/search/msearch-null.json");
+        } catch (ElasticsearchParseException e) {
```

Comment:
If the parse call does not throw, this test will succeed. Isn't the expectation that it should always fail? Should we use `expectThrows` instead?

Reply:
You are right, that's probably what it should be... It throws now (NPE) but there's no guarantee it won't stop throwing in the future, so `expectThrows` is better.